db/migrate/20181112213255_add_level_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLevelToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :level, :integer, default: 0
```

Comment:
Level powinien być raczej dynamicznie obliczany jakąś metodą w modelu user. Skoro level można w sposób deterministyczny wyznaczać na podstawie `experience` to nie ma potrzeby przechowywać go w bazie